evap/staff/importers.py

```diff
@@ -86,18 +87,34 @@ def store_in_database(self, vote_start_date, vote_end_date, semester):
 
 
 class ExcelImporter(object):
+    W_NAME = 'name'
+    W_EMAIL = 'email'
+    W_DUPL = 'duplicate'
+    W_GENERAL = 'general'
+
     def __init__(self, request):
```

Comment:
Yes, I removed django messages as much as possible, they are not only used after redirecting the user when an import was successful.